lib/hydra/pcdm/collection_indexer.rb

```diff
@@ -2,7 +2,9 @@ module Hydra::PCDM
   class CollectionIndexer < ObjectIndexer
     def generate_solr_document
       super.tap do |solr_doc|
-        solr_doc['member_ids_ssim'] = object.member_ids
+        solr_doc['member_ids_ssim'] ||= []
```

Comment:
The string/symbol `member_ids_ssim` is all over the place -- maybe this should be put behind a method somewhere. Also happy to be told, "that's a separate PR."